js/components/PinComment.js

```diff
@@ -21,7 +20,8 @@ export default class PinComment extends React.Component {
 
   _handleSubmit = () => {
     commentPin(
-      Constants.deviceId,
+      "whatever",
```

Comment:
This `"whatever"` is breaking commenting

Reply:
😂